clients/drdisas/drdisas.cpp

```diff
@@ -125,22 +143,55 @@ main(int argc, const char *argv[])
 
     void *dcontext = GLOBAL_DCONTEXT;
 
-#if defined(X86_64) || defined(ARM)
+    dr_disasm_flags_t syntax =
```

Comment:
I found the logic a bit convoluted between all the #ifdef and if-else.
I don't have a better way, but some clarifying comments will help.

Here, something like: "Set the default syntax based on DynamoRIO's build."

Reply:
Done.